### DISCUSSION TITLE
Any replacement for ResultsPerPrompt property?

### DISCUSSION THREAD

**Discussion by Manish-22** [microsoft/semantic-kernel#9042] [Q&A] [answered]
https://github.com/microsoft/semantic-kernel/issues/7053

> The ResultsPerPrompt property has been removed 

Any replacement for this property?

**Comment by evchaki** [comment, marked as answer]
You are correct this is not support - https://learn.microsoft.com/en-us/semantic-kernel/support/migration/v2-openai-migration-guide#3-chatcompletion-multiple-choices-deprecated - getting back multiple choices is not supported by these models any longer.
